src/libs/ReportUtils.js

```diff
@@ -964,6 +964,24 @@ function shouldReportBeInOptionList(report, reportIDFromRoute, isInGSDMode, curr
     return true;
 }
 
+/**
+ * Attempts to find a report in onyx with the provided list of participants
+ * @param {Array} newParticipantList
+ * @param {Array} reports
+ * @returns {Array|undefined}
+ */
+function getChatByParticipants(newParticipantList, reports) {
+    const sortedNewParticipantList = newParticipantList.sort();
+    return _.find(reports, (report) => {
+        // If the report has been deleted, or there are no participants (like an empty #admins room) then skip it
+        if (!report || !report.participants) {
+            return false;
+        }
+        const sortedReportParticipants = report.participants.sort();
+        return sortedNewParticipantList.join() === sortedReportParticipants.join();
```

Comment:
I think `_.isEqual` would be cleaner here

Reply:
👍